lomond/session.py

```diff
@@ -24,10 +24,80 @@
 log = logging.getLogger('lomond')
 
 
+class SelectorBase(object):
```

Comment:
Abstract base class for a 'selector', which has a method to wait until a socket is readable. Python3.3 has a 'selectors' module for this very purpose, but we need Lomond to work on 2.7.